src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Power-Strings.json

```diff
@@ -46,6 +46,14 @@
       "Whenever you remember a new memory, deal #b%1$s damage to ALL enemies and lose all Coil."
     ]
   },
+  "stsjorbsmod:DamnationPower": {
+    "NAME": "Damnation",
+    "DESCRIPTIONS": [
+      "Permanently #yTransform the next card played this combat",
```

Comment:
It's more localization friendly to just have 2 versions of the whole sentence written out, like in the card strings; it's okay for it to be a little redundant, it makes it easier for languages where the more natural phrasing might put the then clause before the object of the sentence